### PR TITLE
Pd 611 gas fee exchange feature

### PR DESCRIPTION
**Pull request by aaroncolaco**

### CODE REVIEW

File: __tests__/integration/transactions.test.ts [hunk 1]
```diff
@@ -770,17 +770,18 @@ describe('Transactions API Integration', () => {
...
-      .reply(200, {
-        safeLow: gasPrice.toString(),
-        standard : gasPrice.toString(),
-        fast: gasPrice.toString(),
-        fastest: '20'
-      })
+    // Appears as though this is never called
+    // const gasPrice = '5'
+    // const GASPRICE_API_HOST = 'https://www.etherchain.org/api/gasPriceOracle'
+    // nock(GASPRICE_API_HOST)
```
Comment (aaroncolaco) [inline __tests__/integration/transactions.test.ts]: This nock intercept in the existing tests was not called and as a result was messing up the new test. Can be tested by ascertained by checking `isDone()` for this one like the coinsuper one is checked.

Spent some time trying to figure out why it was `falsey` but couldn't.

Comment (aaroncolaco) [reply __tests__/integration/transactions.test.ts]: Addressed in the latest commit by clearing out nocks.

File: src/transactions/controller.ts [hunk 2]
```diff
@@ -14,6 +14,19 @@ import validation from '../validation'
 
 const web3 = Config.getPrivateWeb3()
 
+async function getGasFeesInQBX(req, res) {
+  try {
+    const currentGasFees = await qbxFeeCalculator.getGasPrice()
+    const qbxToEthExchangeRate = await qbxFeeCalculator.getQBXToETHExchangeRate()
+    const gasFee = currentGasFees.dividedBy(qbxToEthExchangeRate).toString()
```
Comment (danoctavian) [inline src/transactions/controller.ts]: let's do `toFixed()` here. what happens if you have a big number is it turns it into scientific notation after a certain threshold with toString but toFixed just keeps the whole thing explicit

Comment (aaroncolaco) [reply src/transactions/controller.ts]: Good catch! Done

File: src/transactions/router.ts [hunk 3]
```diff
@@ -51,6 +51,24 @@ const router = express.Router()
  */
 router.get('/raw', LibAPI.wrap(Controller.buildRawTransaction))
 
+/**
+ * @swagger
+ * /transactions/fasfee:
```
Comment (cupaco) [inline src/transactions/router.ts]: gasfee ;)

File: src/transactions/router.ts [hunk 4]
```diff
@@ -51,6 +51,24 @@ const router = express.Router()
...
+ *   get:
+ *     tags:
+ *       - Transactions
+ *     description: Builds a raw transactions ready to be signed
```
Comment (cupaco) [inline src/transactions/router.ts]: Returns the gas fee in QBX? change text.

**Comment by danoctavian** [comment]
based on our conversation we should replace the original endpoint with this:

https://github.com/qiibee/qb-api/pull/64

so this PR as far as i can tell is no longer necessary

**Pull request closed without merging**